Initium-Odp/war/odp/javascript/script.js

```diff
@@ -1481,23 +1483,28 @@ $(document).keyup(function(event){
 		var buttonToPress = $('[shortcut="'+event.which+'"]');
 		if (buttonToPress.length>0)
 		{
-			window.disableShortcuts = true;
+			if (buttonToPress[0].hasClass("main-button-half")==false)
+				window.disableShortcuts = true;
```

Comment:
Disabling shortcuts is kind of an old concept for the full page reloads....
I wonder if it would be better to get rid of it altogether...
On 2016-04-14 3:29 PM, "NikeJoshua" <notifications@github.com> wrote:

> In Initium-Odp/war/odp/javascript/script.js
> <https://github.com/Emperorlou/initium-odp/pull/9#discussion_r59776690>:
>
> > @@ -1481,23 +1483,28 @@ $(document).keyup(function(event){
> >  		var buttonToPress = $('[shortcut="'+event.which+'"]');
> >  		if (buttonToPress.length>0)
> >  		{
> > -			window.disableShortcuts = true;
> > +			if (buttonToPress[0].hasClass("main-button-half")==false)
> > +				window.disableShortcuts = true;
>
> It keeps the shortcuts available if a key for one of the Nearby XXX popups
> was used. Previously, opening one of these popups and then closing it left
> you stuck without shortcuts. I also thought this was a nicer way to do it
> than reenabling on close, as this way you can still use the shortcuts while
> the popup is open (Ex: Done reordering your house inventory? Just press 1
> and you're back in Aera!)
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/Emperorlou/initium-odp/pull/9/files/b5af8284e3f578032b956e57dfe4e6abfb7232ae#r59776690>
>